todo.sh

```diff
@@ -99,6 +99,12 @@ help()
 		        Color mode
 		    -d CONFIG_FILE
 		        Use a configuration file other than the default ~/.todo/config
+		    -l
+		    	Use a (local) todo.txt that is found in the current directory
+		    	or up the directory tree
```

Comment:
I think it should be explicitly mentioned that local vs. global are not mutually exclusive, but that the global todo list will always be used as a fallback:

```suggestion
		    	or up the directory tree, falling back to the configured (global)
		    	TODO_DIR if no local one is found
```